README.md

```diff
@@ -110,17 +110,16 @@ This plugin defines hooks for `environment`, `checkout`, `command`, and `post-co
 
 #### Available properties
 
-- `input_artifacts`: String array of artifact paths to download and sign.
+- `input_artifacts`: String array of artifact paths to download that are required to sign.
+- `sign_prerequisites`: String array of the specific artifact paths to sign. In the case of `.app`s, make sure to list these bottom-up; internal frameworks/helpers first, and `.app` at the end.
+- `output_artifacts`: String array of relevant artifact to upload back to artifacts.
+- `entitlements`: String path of artifact containing entitlements to apply.
```

Comment:
This needs to be a valid plist, right? Not a blocker, but it might make sense to be explicit about exactly what our expected input is here.